docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+2, 3;02.40;ongewijzigd leveren;;;;geslachtsnaam;;;;
+2, 3;03;;;;geboorte;;;;;
+2, 3;03.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+2, 3;03.20;code+omschrijving;;;;plaats;;;;
+2, 3;03.30;code+omschrijving;;;;land;;;;
+2, 3;04.10;code+omschrijving;;;geslacht;;;;;
+2, 3;62.10;ongewijzigd leveren (8 cijfers);;;datumIngangFamilierechtelijkeBetrekking;;;;;
+2, 3;83;;;;inOnderzoek;;;;;
+2, 3;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+2, 3;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+4;;;;nationaliteiten;;;;;;
+4;05.10;code+omschrijving;;;nationaliteit;;;;;
+4;63.10;code+omschrijving;;;redenOpname;;;;;
+4;65.10;code+omschrijving;;;aanduidingBijzonderNederlanderschap;;;;;
+4;83;;;;inOnderzoek;;;;;
+4;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+4;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+4;85.10;ongewijzigd leveren (8 cijfers);;;datumIngangGeldigheid;;;;;
+5;;actuele partner(s) OF de laatst ontbonden partner;;partners;;;;;ontbonden partners worden in GbaPersoon geleverd t.b.v. naamsamenstelling;
+5;01.20;ongewijzigd leveren;;;burgerservicenummer;;;;;
+5;02;;;;naam;;;;;
+5;02.10;ongewijzigd leveren;;;;voornamen;;;;
+5;02.20;code+omschrijving+soort;;;;adellijkeTitelPredicaat;;;;
+5;02.30;ongewijzigd leveren;;;;voorvoegsel;;;;
+5;02.40;ongewijzigd leveren;;;;geslachtsnaam;;;;
+5;03;;;;geboorte;;;;;
+5;03.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+5;03.20;code+omschrijving;;;;plaats;;;;
+5;03.30;code+omschrijving;;;;land;;;;
+5;04.10;code+omschrijving;;;geslacht;;;;;
+5;06;;;;aangaanHuwelijkPartnerschap;;;;;
+5;06.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+5;06.20;code+omschrijving;;;;plaats;;;;
+5;06.30;code+omschrijving;;;;land;;;;
+5;07;;;;ontbindingHuwelijkPartnerschap;;;;;
+5;07.10;ongewijzigd leveren (8 cijfers);;;;datum;;;;
+5;15.10;code+omschrijving;;;soortVerbintenis;;;;;
+5;83;;;;inOnderzoek;;;;;
+5;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+5;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+6;;;;overlijden;;;;;;
+6;08.10;ongewijzigd leveren (8 cijfers);G;;datum;;;;;
+6;08.20;code+omschrijving;;;plaats;;;;;
+6;08.30;code+omschrijving;;;land;;;;;
+6;83;;;;inOnderzoek;;;;;
+6;83.10;ongewijzigd leveren (6 cijfers);;;;aanduidingGegevensInOnderzoek;;;;
+6;83.20;ongewijzigd leveren (8 cijfers);;;;datumIngangOnderzoek;;;;
+7;67;;;opschortingBijhouding;;;;;;
+7;67.10;ongewijzigd leveren (8 cijfers);;;datum;;;;;
+7;67.20;code+omschrijving;G;;reden;;;;;
+7;68.10;ongewijzigd leveren (8 cijfers);;datumEersteInschrijvingGBA;;;;;;
+7;70.10;ongewijzigd leveren (0-7);G;geheimhoudingPersoonsgegevens;;;;;;
+8;09.10;code+omschrijving;G;gemeenteVanInschrijving;;;;;;
+8;09.20;ongewijzigd leveren (8 cijfers);;datumInschrijvingInGemeente;;;;;;
+8;;;;verblijfplaats;;;;;;
+8;10.10;code+omschrijving;G;;functieAdres;;;;;
+8;11.80;ongewijzigd leveren;;;adresseerbaarObjectIdentificatie;;;;;
```

Comment:
zou je deze en de volgende op volgorde categorie-groep-elementnummer kunnen zetten? Dus tussen onder 11.70

Reply:
Is gebeurd.